File: aspnet-core/src/TicketTracker.Application/ProjectUsers/Dto/ProjectUserDto.cs
using Abp.Application.Services.Dto;
using Abp.AutoMapper;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TicketTracker.Entities;
using TicketTracker.Authorization.Users;
using TicketTracker.Users.Dto;
using TicketTracker.ProjectRoles.Dto;

namespace TicketTracker.ProjectUsers.Dto {
    [AutoMap(typeof(ProjectUser))]
    public class ProjectUserDto : EntityDto<int> {
        public int ProjectId { get; set; }
        public SimpleUserDto User { get; set; } 
        public List<PRoleWithPermissionsDto> Roles { get; set; }
    }
}

File: reactjs/src/services/projectUser/dto/projectUserDto.ts
import { EntityDto } from "../../dto/entityDto";
import { PRoleWithPermissionsDto } from "../../projectRole/dto/pRoleWithPermissionsDto";
import { SimpleUserDto } from "../../user/dto/simpleUserDto";

export class ProjectUserDto extends EntityDto { 
    projectId!: number;
    user!: SimpleUserDto;
    roles?: PRoleWithPermissionsDto[];
}